CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 * Refactored `deploydocs` internals to allow dispatch on the `versions` keyword, intended as a non-public API for DocumenterVitepress which cannot use the default versioning mechanism during deployment ([#2695]).
 * Use different banners for dev and unreleased docs ([#2382], [#2682])
+* The search index now loads asynchronously, and can be cached indefinitely. ([#2702], [#2700])
```

Comment:
@xal-0 those PRs also need to be added to the list at the end of the file (make sure to pull our changes first :-) )

Reply:
Note: one can run `make changelog`, which will update the references in `CHANGELOG.md` automatically.